tools/benchmark/src/mocha/memoryTestRunner.ts

```diff
@@ -296,6 +310,17 @@ export function benchmarkMemory(testObject: IMemoryTestObject): Test {
 						heapUsedStats.marginOfErrorPercent > args.maxRelativeMarginOfError
 					);
 
+					const avgHeapUsed = heapUsedStats.arithmeticMean;
+					const allowedMemoryUsage =
+						args.baselineMemoryUsage * (1 + args.allowedDeviation / 100);
+					const tolerance = args.baselineMemoryUsage * (args.allowedDeviation / 100);
+
+					if (ENABLE_MEM_REGRESSION && avgHeapUsed > allowedMemoryUsage) {
```

Comment:
If the calculations are only necessary when `ENABLE_MEM_REGRESSION` is set, it's a bit better to move them inside an if block so they only happen when we're going to use them.